todo_app/data/IndexViewModel.py

```diff
@@ -0,0 +1,69 @@
+from todo_app.data.Item import Item
+from todo_app.data.trello_items import get_trello_lists
+
+class ViewModel:
+    def __init__(self, items, lists, errored):
+        self._items = items
+        self._errored = errored
+        self._sorted_items = {}
+        self._lists = lists
+        self.sort_items()
+
+    def sort_items(self):
+        trello_lists = self.trello_lists
+
+        for trello_list in trello_lists:
+            list_name = trello_list["name"]
+            self._sorted_items[list_name] = []
+
+        #sort items into correct list
+        for item in self._items:
+            self._sorted_items[item.status].append(item)
+
+    @property
+    def sorted_items(self):
+        return self._sorted_items
+
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def trello_lists(self):
+        return self._lists
+
+    @property
+    def errored(self):
+        return self._errored
+
+    @property 
+    def to_do_items(self):
```

Comment:
Is this actually useful? I appreciate you're trying to marry your own take on the exercise with our suggested tests, but it looks like these methods have corresponding tests but they're never used anywhere, so your tests failing doesn't mean the app has problems, and the tests passing doesn't actually indicate anything healthy about the app! We'd ideally ensure we've only got (and are testing) code that is being used

If your template is going to be more dynamic and just iterate over `view_model.sorted_items` then that's great, but that's the function that I'd focus on testing!